docs/framework/deployment/windows/10.md

```diff
@@ -0,0 +1,17 @@
+# Installing the .NET Framework on Windows 10
+
+You may need the .NET Framework to run an application on Windows 10. You can use the following instructions to help you. The instructions depend on which version of the .NET Framework you need.
+
+## .NET Framework 4.6.2
+
+The .NET Framework 4.6.2 is included with the latest version of Windows 10. 
+
+If you are on an earlier version of Windows 10, you are recommended to upgrade to the latest version of [Windows 10](https://www.microsoft.com/software-download/windows10) using Windows Update. You may be able to install the [.NET Framework 4.6.2](https://go.microsoft.com/fwlink/?linkid=845529&source=dotnetdocs) if you are unable to upgrade to the latest version of Windows 10 right now. 
```

Comment:
From our old style guide @GuardRex:
Use first-person plural ("we" and "us") only to refer to Microsoft.